package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "1pr",
+  "version": "1.0.0",
+  "description": "I will be accepting up to one pull request per day on this project.",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/thepracticaldev/1pr.git"
+  },
+  "keywords": [],
+  "author": "",
```

Comment:
I'd be tempted to remove Author or add "thepracticaldev"? 
I'd probably remove keywords too.